src/Core/Numerals/Numbers.fs

```diff
@@ -0,0 +1,13 @@
+﻿module Core.Numerals.Numbers
+
+open Common.StringHelper
+
+type Digit = Digit of int
+type Number1to999 = Number1to999 of int
+type Number1000On = Number1000On of int
```

Comment:
`0O` symbols are a bit confusing put together. is there any other naming idea?

Reply:
How about `NumberFrom1to999` vs `NumberFrom1000`?